src/me/ryanhamshire/AutomaticInventory/AIEventHandler.java

```diff
@@ -369,14 +369,18 @@ public void onPickupItem(PlayerPickupItemEvent event)
             if(firstEmpty < 9) return;
             playerData.firstEmptySlot = firstEmpty; 
             PickupSortTask task = new PickupSortTask(player, playerData, inventory);
-            Bukkit.getServer().getScheduler().scheduleSyncDelayedTask(AutomaticInventory.instance, task, 100L);
+            Bukkit.getServer().getScheduler().scheduleSyncDelayedTask(AutomaticInventory.instance, task, 10L);
         }
     }
 	
 	static void sortPlayerIfEnabled(Player player, PlayerData playerData, Inventory inventory)
 	{
 	    if(featureEnabled(Features.SortInventory, player))
         {
+            // Dont sort inventory if any other inventory is open
+            if (player.getOpenInventory().getTopInventory().getType() != InventoryType.CRAFTING)
```

Comment:
Not even sure if the server knows if this inventory is opened (only knows when it's been closed), but otherwise yea this check would work.